pallets/stateful-storage/src/weights.rs

```diff
@@ -199,6 +204,70 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			.saturating_add(RocksDbWeightChild::get().reads(1_u64))
 			.saturating_add(RocksDbWeightChild::get().writes(1_u64))
 	}
+
+	/// Storage: `Msa::PublicKeyToMsaId` (r:1 w:0)
+	/// Proof: `Msa::PublicKeyToMsaId` (`max_values`: None, `max_size`: Some(48), added: 2523, mode: `MaxEncodedLen`)
+	/// Storage: `Schemas::IntentInfos` (r:1 w:0)
+	/// Proof: `Schemas::IntentInfos` (`max_values`: None, `max_size`: Some(13), added: 2488, mode: `MaxEncodedLen`)
+	/// Storage: `Schemas::SchemaInfos` (r:23 w:0)
+	/// Proof: `Schemas::SchemaInfos` (`max_values`: None, `max_size`: Some(17), added: 2492, mode: `MaxEncodedLen`)
+	/// Storage: UNKNOWN KEY `0x0763c98381dc89abe38627fe2f98cb7af1577fbf1d628fdddb4ebfc6e8d95fb1` (r:1 w:1)
+	/// Proof: UNKNOWN KEY `0x0763c98381dc89abe38627fe2f98cb7af1577fbf1d628fdddb4ebfc6e8d95fb1` (r:1 w:1)
+	fn delete_page_with_signature_v3() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `2112`
+		//  Estimated: `60286`
+		// Minimum execution time: 168_000_000 picoseconds.
+		Weight::from_parts(175_000_000, 60286)
+			.saturating_add(T::DbWeight::get().reads(26_u64))
```

Comment:
This does not make sense. Reading 23 schemaInfos ?

Reply:
Some of these DB weights should be replaced with RocksDbWeightChild. We need to rerun the weights on this.